### PR TITLE
chore(debug-migration): trigger migration for teamsfx tasks and commands

### PR DESCRIPTION
**Pull request by kuojianlu**

### CODE REVIEW

File: packages/vscode-extension/src/debug/teamsfxTaskProvider.ts [hunk 1]
```diff
@@ -108,6 +108,16 @@ const customTasks = Object.freeze({
   },
 });
 
+async function triggerV3Migration(): Promise<void> {
```
Comment (swatDong) [inline packages/vscode-extension/src/debug/teamsfxTaskProvider.ts:111]: Why defining `triggerV3Migration()` again?

Comment (kuojianlu) [reply packages/vscode-extension/src/debug/teamsfxTaskProvider.ts:111]: They are a little different, i.e., different return values.

Comment (swatDong) [reply packages/vscode-extension/src/debug/teamsfxTaskProvider.ts:111]: Yep... that's why I asked... if the only difference is return value, still can reuse the above one - just ignore the return value.

Comment (kuojianlu) [reply packages/vscode-extension/src/debug/teamsfxTaskProvider.ts]: You're right, updated.

**Review by XiaofuHuang — APPROVED** [review]

**Review by HuihuiWu-Microsoft — APPROVED** [review]

**Pull request merged**